### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by rhnonose**
- Extract queries into own module
- Refactor some code

### CODE REVIEW

File: lib/re/listings/queries.ex [hunk 1]
```diff
@@ -0,0 +1,12 @@
+defmodule Re.Listings.Queries do
```
Comment (gusaiani) [inline lib/re/listings/queries.ex]: Modules should have a @moduledoc tag.

File: lib/re/images/queries.ex [hunk 2]
```diff
@@ -0,0 +1,19 @@
+defmodule Re.Images.Queries do
```
Comment (gusaiani) [inline lib/re/images/queries.ex]: Modules should have a @moduledoc tag.

**Comment by gusaiani** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 2 possible new issues (including those that may have been commented here).
* 1 fixed issue! 🎉

You can see more details about this review at https://ebertapp.io/github/em-casa/backend/pulls/144.

**Comment by gusaiani** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 1 fixed issue! 🎉

You can see more details about this review at https://ebertapp.io/github/em-casa/backend/pulls/144.

**Comment by rhnonose** [comment]
Sounds fair, I'll think about the names.

To be honest, I'm not sure if `Queries` was a good choice for the module name, but can't think of a better one for now.

**Comment by gusaiani** [comment]
@rhnonose 
Thanks!
I like `Queries` well enough ;)

**Pull request merged**